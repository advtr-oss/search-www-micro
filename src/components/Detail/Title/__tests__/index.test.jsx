/**
 * @jest-environment jsdom
 */

import React from 'react'
import { render as rtlRender } from '@testing-library/react'
import { GlobalStyle, ThemeProvider } from '@advtr/tidy'

import { Title } from '../index'

const render = (ui, opts) => rtlRender(ui, {
  ...opts,
  wrapper: ({ children }) => (
    <ThemeProvider colorScheme='light'>
      <>
        <GlobalStyle />
        {children}
      </>
    </ThemeProvider>
  )
})

describe('Detail.Title', function () {
  it('should render a snapshot', function () {
    const { container } = render(<Title secondary='Halifax' primary='Calderdale, United Kingdom' />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
