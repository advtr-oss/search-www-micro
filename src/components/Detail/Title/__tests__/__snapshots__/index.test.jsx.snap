// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail.Title should render a snapshot 1`] = `
.c1 {
  color: #57606a !important;
}

.c0 .primary {
  margin: 0;
}

.c0 .secondary {
  margin: 4px 0;
}

<div
  class="c0"
>
  <h3
    class="h3 primary"
  >
    Calderdale, United Kingdom
  </h3>
  <h4
    class="c1 h4 secondary"
  >
    Halifax
  </h4>
</div>
`;
